File: LPS_Angular/src/app/models/User/UserRoleModel.ts
export class UserRoleModel {
    STAFF_NO: string;
    ROLE_CODE: string;
    ROLE_DESC: string;
    AO_KEY: string;
    ROLE_LEVEL: number;
}

export class RoleOrgModel {
    ROLE_CODE: string;
    ROLE_DESC: string;
    ROLE_PARENT: string;
    ROLE_LEVEL: number;
    ROLE_PARENT_DESC: string;
}

export class GetRoleOrgModel {
    department: RoleOrgModel[];
    subDepartment: RoleOrgModel[];
    unit: RoleOrgModel[];
    unitLeader: RoleOrgModel[];
}

export enum RoleType {
    Security = 0,
    Rm = 1
}

File: LPS_Service/Models/User/UserRoleModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace LPS_Service.Models.User
{
    public class UserRoleModel
    {
        public string STAFF_NO { get; set; }
        public string ROLE_CODE { get; set; }
        public string ROLE_DESC { get; set; }
        public string AO_KEY { get; set; }
        public int? ROLE_LEVEL { get; set; }
    }
}